src/components/ActionBar.vue

```diff
@@ -620,16 +622,16 @@ div.bar-filter-wrapper
           @clickBtn="discardAllFilter()"
           fontSize='15'
         )
-      <!--div.btn-320-wrapper.row.items-center.scroll-anim-->
-        <!--btn.create-poll-btn.btn-320(-->
-          <!--iconRight-->
-          <!--:labelText="$t('pages.trails.ballots.actionBar.btnCreatePoll')"-->
-          <!--primary-->
-          <!--btnWidth='155'-->
-          <!--fontSize='16'-->
-          <!--padding='0'-->
-          <!--@clickBtn="isAuthenticated ? openBallotForm() : openNotice()"-->
-        <!--)-->
+      div.btn-320-wrapper.row.items-center.scroll-anim
```

Comment:
We also had this one removed on purpose #102

Reply:
I did rebase the branch with develop after you merged the [PR#123](https://github.com/telosnetwork/app-telos-native/pull/123)